BONGWE.srt

```diff
@@ -4,37 +4,36 @@
 
 2
 01:00:03,625 --> 01:00:10,000
-<b>Our problem here at Bongwe village is water.We dont go in </b>
+<b>Our problem here at Bongwe Village is water. We don't go in</b>
 <b>our fields in the morning and we usually go at the afternoon</b>
 
 3
 01:00:10,000 --> 01:00:18,791
 <b>as you can see, we struggle for water. When our children </b>
-<b>come from school they Nsima very late at 16:00.When this </b>
-<b>well dry up  we fetch water at Dzombe</b>
+<b>come from school they Nsima very late at 16:00. When this</b>
+<b>well dry up we fetch water at Dzombe</b>
 
 4
 01:00:18,791 --> 01:00:32,958
-<b>Chaula, Chipeni, Nkhoma and villages.We all depend on this </b>
-<b>well.Women fight over water.</b>
+<b>Chaula, Chipeni, Nkhoma and villages all depend on this</b>
+<b>well. Women fight over water.</b>
 
 5
 01:00:33,125 --> 01:00:45,291
-<b>Its hard for us to know that we are in Malawi.It is like we </b>
-<b>are in a foregn country.We always get a half bucket so that </b>
+<b>It's hard for us to know that we are in Malawi. It is like we </b>
+<b>are in a foregn country. We always get a half bucket so that</b>
 <b>every one can drink water.</b>
 
 6
 01:00:45,333 --> 01:00:57,583
-<b>Am begging our friends in health please help us.As this well</b>
-<b> dries up we go on the other dirty well.Its very pathetic </b>
+<b>Am begging our friends in health please help us. As this well</b>
+<b>dries up we go on the other dirty well. It is very pathetic.</b>
 
 7
 01:00:57,583 --> 01:01:08,208
-<b>We dont sleep in the house but outside and we meet hynas and</b>
-<b> we will be bitten by the hynas please help  us.</b>
+<b>We don't sleep in the house but outside and we meet hyenas.</b>
```

Comment:
Why do they sleep outside? Does this have to do with the water? Do hyenas really just come up and bite sleeping people? Just trying to understand this.